wildfly/commit-markable-resource/pom.xml

```diff
@@ -38,32 +38,11 @@
     <url>http://narayana.io</url>
 
     <properties>
-        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <maven.compiler.target>1.8</maven.compiler.target>
-        <maven.compiler.source>1.8</maven.compiler.source>
-        <failOnMissingWebXml>false</failOnMissingWebXml>
-
         <server.args>-Dprojectdir=${project.build.directory}</server.args>
-        <surefire.argline></surefire.argline>
+        <surefire.argline>-Djdk.attach.allowAttachSelf=true</surefire.argline>
```

Comment:
Aren't you declaring this in `arquillian.xml`? I am a bit confused with all these repetitions, are they needed?

Reply:
Thanks @jmfinelli ! Yes, this is a repetition. I am going to restore the old line. Actually it is not taken from arquillian.xml but it is taken from this profile: https://github.com/jbosstm/quickstart/blob/master/wildfly/commit-markable-resource/pom.xml#L250-L263
As written in the comment:  " Byteman-Arquillian-related profiles for configuring use of Byteman on different platforms/JDKs  these setup is needed for Arquillian Extension Byteman which makes possible to use Byteman rules with @BMRule while Arquillian JUnit runner is in command "